incident/models/incident_index_page.py

```diff
@@ -235,7 +235,7 @@ def get_context(self, request, *args, **kwargs):
         # Could also be:
         # context['incident_count'] = incident_qs.count()
 
-        if request.is_ajax():
+        if request.headers.get('x-requested-with') == 'XMLHttpRequest':
```

Comment:
Same comment here as for the blog index page. I basically think it's safe to remove setting this context variable to anything: `layout_template` doesn't seem like it's used anywhere and the `base.ajax.html` is not present anymore (as far as I can tell).